lib/Target/HSACO/HSACOTranslation.cpp

```diff
@@ -116,7 +116,7 @@ initialize_module(llvm::Module *module, const std::string &triple,
   opt.NoNaNsFPMath = true;
   llvm::TargetMachine *machine = target->createTargetMachine(
       module->getTargetTriple(), proc, features, opt, llvm::Reloc::PIC_,
-      std::nullopt, llvm::CodeGenOpt::Aggressive);
+      std::nullopt, llvm::CodeGenOptLevel::Aggressive);
```

Comment:
@micmelesse Can you review changes in this file? It's due to recent change of LLVM.